File: data/EXP01/python/2221/2221.output_3.py
# @lc app=leetcode id=2221 lang=python3
from typing import List

class Solution:
    def triangularSum(self, nums: List[int]) -> int:
        while len(nums) > 1:
            newNums = []
            for i in range(len(nums) - 1):
                newNums.append((nums[i] + nums[i+1]) % 10)
            nums = newNums
        return nums[0]

File: data/EXP01/golang_no_system_var/2221/2221.output_3.go
func triangularSum(nums []int) int {
    n := len(nums)

    for n > 1 {
        newNums := make([]int, n-1)
        for i := 0; i < n-1; i++ {
            newNums[i] = (nums[i] + nums[i+1]) % 10
        }
        nums = newNums
        n = len(nums)
    }

    return nums[0]
}
